src/schnetpack/representation/painn.py

```diff
@@ -248,28 +233,10 @@ def forward(self, inputs: Dict[str, torch.Tensor]):
             filter_list = torch.split(filters, 3 * self.n_atom_basis, dim=-1)
 
         # compute initial embeddings
-        q = self.embedding(atomic_numbers)[:, None]
-        
-        # add spin and charge embeddings
-        if hasattr(self, "activate_charge_spin_embedding") and self.activate_charge_spin_embedding:
-            # get tensors from input dictionary
-            total_charge = inputs[properties.total_charge]
-            spin = inputs[properties.spin_multiplicity]
-            num_batch = len(inputs[properties.idx])
-            idx_m = inputs[properties.idx_m]
-
-            charge_embedding = self.charge_embedding(
-                q.squeeze(),
-                total_charge,
-                num_batch,
-                idx_m
-            )[:, None]
-            spin_embedding = self.spin_embedding(
-                q.squeeze(), spin, num_batch, idx_m
-            )[:, None]
-
-            # additive combining of nuclear, charge and spin embedding
-            q = (q + charge_embedding + spin_embedding)
+        q = self.embedding(atomic_numbers)
+        for embedding in self.interactions:
```

Comment:
this should be self.electronic_embeddings instead of self.interactions. correct?